{
  try {
    XQException ex=new XQException("Hello world!");
    junit.framework.Assert.assertEquals("A-XQEX-1.1: XQException successfully created","Hello world!",ex.getMessage());
  }
 catch (  Exception e) {
    junit.framework.Assert.fail("A-XQEX-1.1: Creating XQException failed with message: " + e.getMessage());
  }
}
